src/core/dbus-execute.c

```diff
@@ -375,6 +375,10 @@ static int property_get_syscall_filter(
                 char *s;
                 int num = PTR_TO_INT(val);
 
+                if (c->syscall_allow_list && num >= 0)
```

Comment:
This looks like a backward incompatible change? And also I'm not sure it's right? It means nothing will be returned when it's allow-listed